addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9170,8 +9170,10 @@ msgctxt "#15310"
 msgid "Opening multi-path source"
 msgstr ""
 
+#. Label for the path infolabel which shows the location of the media file
+#: xbmc/addons/skin.estuary/xml/Includes_MusicInfo.xml
```

Comment:
this should not be done here but into skin language file

Reply:
i mean since it is being recreated, it might be considered to move it to the appropriate skin language file

Reply:
not really. if this is a rather generic string with a fixed semantics like in this case it is okay to have it in kodi core strings for reuse by add-ons to avoid "each and every" addon having it to define itself. This is how we have done it since forever. Problem with this approach is as we can see in this PR are changes of those strings because you never know whether something will break for somebody as you cannot say for sure where this string is used. Core strings have to be considered as part of Kodi API and every string change is a breaking change, strictly speaking.

Not saying that the current concept is good, just saying how we did so far and that there are no other rules in place.

Reply:
yeah, having "core" strings that can be used everywhere at least in my opinion is not very good,
because they can be used by any addon or also on c++ sources

so imo if you are working on text to be used for skinning it should be limited to that purpose on his resources without using "core" strings that can affect anything

at least that's how i see it

Reply:
Without a team decision (which you could request team internally) we will end up nowhere here I fear. No rules in place => everybody can to what they want.

Reply:
no problem. i pointed this out because in the past @enen92 he asked me to operate in this way for new changes

Reply:
To me punctuation marks should never be part of a generic string as it limits the reuse, and from a search of the strings files this is indeed the only one like this. If punctuation marks are thought be be required then it should be done in core, the skin, or addon.

So I was expecting this to be a simply uncontroversial correction by simply removing the punctuation mark while the actual string element is unchanged. Although of course I know that still invalidates all the existing translations.

However I understand the points being raised here so I think it best to close this and for @Hitcher to add a new string to the Estuary language file when he submits the info dialog changes.